persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/TestEntity.scala

```diff
@@ -43,9 +43,9 @@ object TestEntity {
     override def aggregateTag: AggregateEventShards[Evt] = Evt.aggregateEventShards
   }
 
-  final case class Appended(element: String) extends Evt
+  final case class Appended(entityId: String, element: String) extends Evt
```

Comment:
yes, I removed it from the TestEntity because I don't think it should be needed in the events, but I might have missed something of the new things in the read side. Let's see what James says.